src/gui_node.rs

```diff
@@ -143,32 +132,34 @@ impl Widget for GuiNode {
                             output_event = Event::Click;
                         }
                     }
-                    conrod::event::Widget::Press(press) => {
-                        match press.button {
-                            conrod::event::Button::Mouse(_, _) => {
-                                if press.modifiers.contains(conrod::input::keyboard::CTRL) {
-                                    data.mode = Mode::OutputConnection;
-                                    output_event = Event::ConnectOutput;
-                                } else {
-                                    data.mode = Mode::Drag;
-                                }
+                    conrod::event::Widget::Press(press) => match press.button {
```

Comment:
This lifting of a nested match term up one level is also something rustfmt does by default.